example/rust-example.md

```diff
@@ -0,0 +1,75 @@
+# Prerequisites
+
+There need rust toolchain in your local environment, if you haven't set it up, you can refer [here](https://www.rust-lang.org/tools/install) to figure it out.
```

Comment:
It's better refer to https://github.com/apache/incubator-opendal/blob/main/CONTRIBUTING.md instead